src/app/analyze/analyze-routing.module.ts

```diff
@@ -4,6 +4,8 @@ import { RouterModule, Routes } from '@angular/router';
 import { AnalyzeComponent } from './analyze.component';
 import { AnalyzeOverviewComponent } from './analyze-overview/analyze-overview.component'
 
+import {RenderStackDetailsComponent} from './stackreports/render-stack-details/render-stack-details.component';
```

Comment:
You need to keep the stack details stuff as a module like I did. Please can you remove the changes from this file